### PR TITLE
BaSP-TG-10:Task

### PR DESCRIPTION
**Pull request by julianpiaggio**

### CODE REVIEW

**Review by leimeter-joaquin — CHANGES_REQUESTED** [review]
Please add the changes to the  postman collection

File: src/controllers/tasks.js [hunk 1]
```diff
@@ -1,118 +1,70 @@
...
+      message: `An error ocurred: ${error.message}`,
+      error: true,
+    });
   }
-
-  res.status(200).json({ filteredTasks });
 };
 
-module.exports = {
+export default {
```
Comment (leimeter-joaquin) [inline src/controllers/tasks.js:66]: Don't export default, export every function individually.

File: src/routes/tasks.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+import taskControllers from '../controllers/tasks';
```
Comment (leimeter-joaquin) [inline src/routes/tasks.js:2]:
```suggestion
import { getAllTasks, getTasksById, createTask } from '../controllers/tasks';
```

**Review by leimeter-joaquin — CHANGES_REQUESTED** [review]

**Review by leimeter-joaquin — DISMISSED** [review]

**Review by kolodani — APPROVED** [review]
great work, ready to merge

**Pull request merged**